profile-diff/style.css

```diff
@@ -0,0 +1,352 @@
+  background-color: #f8fafd;
+}
+
+.header {
+  height: 7.25em;
+  background-color: #233876;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 1.25em;
+  color: var(--white);
+  position: relative;
+}
+
+.back {
+  background-color: transparent;
+  border: none;
+  padding: 0.625em;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  position: absolute;
+  left: 10%;
+}
+
+.back__icon {
+  height: 1.6rem;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 2em 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1em;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: x-large;
+  text-align: center;
+  width: 95%;
+  max-width: 992px;
+}
+
+.user_image {
+  border-radius: 37px;
+}
+
+.user_name {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 150%;
+  color: #111928;
+  margin-top: -1em;
+  margin-bottom: 1em;
+}
+
+.expand-control-container {
+  width: 100%;
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  color: #233876;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+.expand-control-text {
+  cursor: pointer;
+}
+
+.expand-control-icon {
+  margin-left: 0.5em;
+  font-size: 10px !important;
+}
+
+.reason_heading {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 150%;
+  color: #111928;
+  width: 100%;
+  text-align: left;
+  padding-left: 1.25em;
+  margin-block-end: 0;
+}
+
+.reason_textarea {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  background: #ffffff;
+  box-shadow: 0px 8px 3px rgba(0, 0, 0, 0.01), 0px 5px 3px rgba(0, 0, 0, 0.02),
+    0px 2px 2px rgba(0, 0, 0, 0.04), 0px 1px 1px rgba(0, 0, 0, 0.04);
+  border-radius: 12px;
+  border: none;
+  padding: 1em;
+  width: 100%;
+  resize: none;
+}
+
+.button__container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 1em;
+}
+
+.button__approve {
+  padding: 11px 32px;
+  background: #046c4e;
+  border-radius: 10px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 150%;
+  color: #ffffff;
+  cursor: pointer;
+}
+
+.button__approve:disabled {
+  opacity: 0.2;
+}
+
+.button__reject {
+  padding: 11px 32px;
+  background: #c81e1e;
+  border-radius: 10px;
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 150%;
+  color: #ffffff;
+  cursor: pointer;
+}
+
+.button__reject:disabled {
+  opacity: 0.2;
+}
+
+.loader {
+  width: 60px;
+  aspect-ratio: 2;
+  --_g: no-repeat radial-gradient(circle closest-side, #000 90%, #0000);
```

Comment:
- radial gradient in a loader? can you add a screenshot of how this looks please?